src/styles/mix.scss

```diff
@@ -0,0 +1,9 @@
+.hand {
+  position: absolute;
+  bottom: 50%;
+  transform-origin: bottom;
+}
+
+.watch {
+  position: relative;
+}
```

Comment:
why it is in `mix` file, what's the point of it? if it is a mix, just add some another naming, for ex:
```html
<div class="watch watch-primary">
  ...
</div>
```

and then:
```scss
.watch-primary {
  position: relative;
}
```